src/sdk/tests/integration/TopicMessageSubmitTransactionIntegrationTests.cc

```diff
@@ -317,9 +318,10 @@ TEST_F(TopicMessageSubmitTransactionIntegrationTests, CannotSubmitTopicMessageWi
 
   // When
   std::vector<TransactionResponse> txResponses;
-  EXPECT_NO_THROW(
+  EXPECT_THROW(
     txResponses =
-      TopicMessageSubmitTransaction().setMessage(getTestBigContents()).setMaxChunks(15).executeAll(getTestClient()));
+      TopicMessageSubmitTransaction().setMessage(getTestBigContents()).setMaxChunks(15).executeAll(getTestClient()),
+    PrecheckStatusException);
```

Comment:
If this fails at precheck now we can probably remove the checks for `ReceiptStatusException` under this?

Reply:
I will leave this as is for now because from a couple of service versions that came the tests where a bit flaky. One time it was `ReceiptStatusException` on the next version was `PrecheckStatusException` and this changed a few times -.-